src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+// actions
+
+const CHECK = 'CHECK';
```

Comment:
- [ ] I kindly suggest that the value of each action type should follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).

## EXAMPLE
```
const CHECK_STATUS  = bookstore/categories/CHECK_STATUS

```

Reply:
Thanks for the suggestion, done!